Assets/Samples/Continuous/ContinuousDemo.cs

```diff
@@ -23,6 +30,8 @@ public class ContinuousDemo : MonoBehaviour {
 		BarcodeScanner.OnReady += (sender, arg) => {
 			// Set Orientation & Texture
 			Image.transform.localEulerAngles = new Vector3(0f, 0f, BarcodeScanner.Camera.GetRotation());
+			float scaleY = BarcodeScanner.Camera.IsVerticalyMirrored() ? -1f : 1f;
```

Comment:
hum yes and no ...

Yes I'm agree, the camera can provides better accessors, like directly providing a EulerAngles or a Scale Vector3 for peoples who don't want to do math.

Something like :
```csharp
Image.transform.localEulerAngles = BarcodeScanner.Camera.GetTextureEulerAngles();
Image.transform.localScale = BarcodeScanner.Camera.GetTextureScale();
```

But the size or the orientation of a game object should remains to the implementation, a camera shouldn't enforce how his texture is used or displayed on the screen.

And your ideal world with the texture automatically returned with a correct angle / direction ... you are asking for magic here.
This really depends of your implementation and what you consider as "the correct angle". And it's without even saying that rotating a object is really easy (purely done by GPU on a single draw call), but rotating a texture internally is a really heavy operation for the CPU ^^